sentry-tower/src/helloworld.rs

```diff
@@ -0,0 +1,209 @@
+/// The request message containing the user's name.
```

Comment:
So this is auto-generated from a grpc definition somehow, for usage in the doctest via `include!`. Can you add a comment on top explaining this, and also give the original definition with which to re-generate this file if necessary?

Reply:
Yeah, this is straight off the usual greeter proto example files, in this case from the `tonic` examples.

Link added to it in 110471b